mathjax3-ts/output/chtml/Wrappers.ts

```diff
@@ -0,0 +1,34 @@
+/*************************************************************
```

Comment:
Just for clarification: Wrappers combines all wrapper implementations for the different node types.

Reply:
Wrappers is an object that indexes all the wrappers for the different node types, yes.  It is used by an instance of a  WrapperFactory in order to know what types of wrappers it can create.